### DISCUSSION TITLE
Hover preview

### DISCUSSION THREAD

**Discussion by YQisme** [squidfunk/mkdocs-material#5727] [Ideas]
Can you consider this new feature? Hovering the mouse over the link can preview the content, just like Wikipedia and Obsidian.

![image](https://github.com/squidfunk/mkdocs-material/assets/80308783/af48bd6c-7870-41ba-94ff-54ebe3cbb49a)

![image](https://github.com/squidfunk/mkdocs-material/assets/80308783/febce904-ff2a-483f-bbcd-77d8122eb30b)

**Comment by squidfunk** [comment]
Jep, support is planned, but we haven't had the chance to finish the implementation yet. [Here's a prototype](https://twitter.com/squidfunk/status/1466794654213492743?lang=en).

**Reply by YQisme** [reply]
Great, that's what I want. This is a cool and convenient feature that I can't wait for.
 Can you please let me know when it will be scheduled? After all, it has been almost two years.

**Reply by squidfunk** [reply]
We'll start working on it once the next funding goal is hit. The background is that the next funding goal will release tooltips into the community edition, and we're always using this release cycle to do some refactoring. Tooltips are the basic building block of the instant preview functionality, so after we had a chance to refactor them, we'll look into adding support for instant preview shortly.

We wanted to work on this feature earlier, but you know how it sometimes is: more important things come in the way, so we needed to reschedule. We expect the next funding goal to be hit later this year, and instant previews to be added a few weeks after that.

Currently, all focus is on the release of the blog plugin into the community edition, an entirely new search implementation that is written from scratch, and a restructuring of our documentation to better account for the many options that Material for MkDocs has. All of this together + community and sponsor management is quite a full-time job. With more funding through sponsorships, we hope to on-board further bright devs to help us push the project forward 😊

**Reply by YQisme** [reply]
Thank you for your detailed reply.

**Comment by rafik-mammeri** [comment]
I found this discussion while looking for a way to make such preview using the annotation system, specifically I wanted to have a preview of the content of an admonition by clicking on a link to it. My idea was to get the content of the admonition using a hook then annotate text. I don't know if it's possible to add some custom "templating" to annotate text.

To give some context my goal is to write maths documents in latex like style by creating custom admonitions to define Theorems, Lemmas, Definitions...etc. I want to be able to make references to these custom admonitions without making a direct link but by showing the content of the admonition by clicking on the "reference" like annotations work

The workaround I will use is to make the annotation containing the admonition content right after the link. It does the job but its not optimal.

Looking forward for this new feature
